x/axelarnet/keeper/msg_server.go

```diff
@@ -290,31 +290,35 @@ func (s msgServer) RouteIBCTransfers(c context.Context, req *types.RouteIBCTrans
 	for _, c := range s.BaseKeeper.GetCosmosChains(ctx) {
 		chain, ok := s.nexus.GetChain(ctx, c)
 		if !ok {
-			ctx.Logger().Error(fmt.Sprintf("%s is not a registered chain", chain.Name))
+			s.Logger(ctx).Error(fmt.Sprintf("%s is not a registered chain", chain.Name))
+			continue
+		}
+
+		if chain.Name == types.ModuleName {
```

Comment:
Why not use `exported.Axelarnet.Name` since we're checking the chain name. The module name should be independent of that (even though it happens to be the same rn).

Reply:
it used ModuleName in default genesis

Reply:
Sounds like we should fix that instead

Reply:
We can do that refactoring for next week's upgrade I guess.